src/FINDOLOGIC/Api/Responses/Xml21/Properties/Servers.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Servers
+{
+    /** @var string $frontend */
+    private $frontend;
+
+    /** @var string $backend */
+    private $backend;
+
+    /**
+     * Servers constructor.
+     * @param SimpleXMLElement $response
+     */
+    public function __construct($response)
```

Comment:
Please typehint this with `SimpleXMLElement` and drop the PHPDoc as it is not really needed anyway.